src/Repository/ProjectRepository.php

```diff
@@ -5,51 +5,50 @@
 namespace App\Repository;
 
 use App\Entity\Project;
-use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
-use Doctrine\Persistence\ManagerRegistry;
-
-/**
- * @method Project|null find($id, $lockMode = null, $lockVersion = null)
- * @method Project|null findOneBy(array $criteria, array $orderBy = null)
- * @method Project[] findAll()
- * @method Project[] findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
- *
```

Comment:
This is Symfony at its worst :/ 
I realize it's part of default generator and Symfony Demo. So don't worry about it